Source/MediaInfo/Multiple/File_Nsv.cpp

```diff
@@ -576,7 +576,7 @@ void File_Nsv::Data_Parse()
 
                 Stream_Prepare(Stream_Type[i]);
                 Fill(Stream_Type[i], 0, Fill_Parameter(Stream_Type[i], Generic_CodecID), Ztring().From_CC4(Stream.codecid));
-                File__Analyze* Parser;
+                File__Analyze* Parser=NULL;

```

Comment:
I prefer a default: Parser=NULL; at the end of the switch. It will never happen but the the compiler is not aware about it :).